common/djangoapps/util/tests/test_db.py

```diff
@@ -198,6 +198,10 @@ class MigrationTests(TestCase):
     """
 
     @override_settings(MIGRATION_MODULES={})
+    @unittest.skip(
+        'Temporarily skipping until the last release of https://openedx.atlassian.net/browse/ENT-2892 in which'
+        'we rename two columns.
```

Comment:
missing the closing single quote

Reply:
pffft, what's a little missing syntax between friends??

Reply:
😂